vector_icons/chrome/app/vector_icons/default_favicon.icon

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This icon doesn't seem to be working @rossmoody - where did you intend it to go - on the tab?
![image](https://user-images.githubusercontent.com/741836/46494375-8fd01c00-c7c7-11e8-95d7-781c754a8b04.png)

Reply:
So looks like this icon is only included on the macOS build, possibly related to touchbar, though I couldn't get it to show up. Let's keep it in, but if we also want to change the default favicon, I think we'll need to provide PNGs @rossmoody . I'll create a separate issue

Reply:
https://github.com/brave/brave-browser/issues/1433